docs/src/modules/ROOT/pages/optimization-algorithms/overview.adoc

```diff
@@ -914,23 +908,6 @@ add the `customProperties` element and use xref:using-timefold-solver/configurat
 ----
 
 
-[#noChangeSolverPhase]
```

Comment:
I'm curious. Which rare cases would it be?

Reply:
I can not come up with any, which is why I've deprecated it for removal.
I'm guessing it's something ancient that is no longer necessary.

If people notice the deprecation, they may come back to us and tell us. At which point, we can decide if this is something we want to support, and then perhaps figure out a proper way of solving the use case.